ipcontroller_config.py

```diff
@@ -238,7 +238,7 @@
 
 # The IP address for registration.  This is generally either '127.0.0.1' for
 # loopback only or '*' for all interfaces.
-# c.HubFactory.ip = u''
```

Comment:
In other words, here I put in an explicit ip address for the front end node, and the registration process completes:

2015-05-06 17:01:37.980 [IPEngineApp] Completed registration with id 3
2015-05-06 17:01:37.982 [IPEngineApp] Completed registration with id 0
2015-05-06 17:01:37.982 [IPEngineApp] Completed registration with id 2
...